docs/pages/connect-your-client/request-access.mdx

```diff
@@ -0,0 +1,174 @@
+   ```
+
+1. Request access to the resource by running the following command, inputting
+   the resource ID you found in the previous step:
+
+   ```code
+   $ tsh request create --resource RESOURCE_ID
+   ```
+
+### Kubernetes resources
+
+In addition to Teleport-protected Kubernetes clusters, you can request access to
+resources within a Kubernetes cluster. In this case, the procedure is slightly
+different than requesting access to a Teleport-protected infrastructure
+resource.
+
+1. Search for a resource to request access to:
+
+   ```code
+   $ tsh request search --kind=<KIND>
+   ```
+
+   Substitute `--kind` with the kind of Teleport resource you would like to
+   request access to. 
+
+   Kubernetes resources are scoped either to a cluster or a namespace.
+
+   You can specify the following namespaced resources: `pod`, `secret`,
+   `configmap`, `service`, `serviceaccount`, `persistentvolumeclaim`,
+   `deployment`, `replicaset`, `statefulset`, `daemonset`, `kube_role`,
+   `rolebinding`, `cronjob`, `job`, `ingress`.
+
+   Teleport supports the following cluster-wide resources: `namespace`,
+   `kube_node`, `clusterrole`, `clusterrolebinding`, `persistentvolume`,
+   `certificatesigningrequest`.
+
+1. Determine the ID of the resource to request access to. 
+
+   For Kubernetes namespaced resources, the `resource-id` is in the following format:
+
+   ```
+   /TELEPORT_CLUSTER/NAMESPACED_KIND/KUBE_CLUSTER/NAMESPACE/RESOURCE_NAME
+   ```
+
+   For example, a pod called `nginx-1` in the `development` namespace of the
+   `mycluster` cluster would have the following ID:
+
+   ```
+   /teleport.example.com/pod/mycluster/development/nginx-1
+   ```
+
+   For Kubernetes cluster-scoped resources, the `resource-id` is in the following format:
+   
+   ```
+   /TELEPORT_CLUSTER/CLUSTER_WIDE_KIND/KUBE_CLUSTER/RESOURCE_NAME
+   ```
+
+   For example, the ID of a namespace (a cluster-scoped resource) called `prod` in the cluster `mycluster`
+   would be:
+
+   ```
+   /teleport.example.com/namespace/mycluster/prod
+   ```
+
+1. Request access to the resource by running the following command, inputting
+   the resource ID you determined in the previous step.
+
+   ```code
+   $ tsh request create --resource RESOURCE_ID
+   ```
+
+   For example, to request access to a pod called `nginx-1` in the `development`
+   namespace, run the following command:
+
+   ```code
+   $ tsh request create --resource /teleport.example.com/pod/mycluster/development/nginx-1
+   ```
+
+   For the `NAMESPACE` and `RESOURCE_NAME` values, you can match ranges of
+   characters by supplying a wildcard (`*`) or regular expression. Regular
+   expressions must begin with `^` and end with `$`.
+
+   For example, to create a request to access all pods in all namespaces that
+   match the regular expression `/^nginx-[a-z0-9-]+$/`, run the following
+   command:
+
+   ```code
+   $ tsh request create --resource /teleport.example.com/pod/mycluster/*/^nginx-[a-z0-9-]+$
```

Comment:
nit: the resource id should be quoted to avoid * , [] and $ to be evaluated by the shell.